src/redisearch_rs/inverted_index_bencher/src/ffi.rs

```diff
@@ -102,10 +102,69 @@ pub fn read_freqs(buffer: &mut Buffer, base_id: u64) -> (bool, inverted_index::R
     (returned, result)
 }
 
+pub fn encode_freqs_fields(
+    buffer: &mut TestBuffer,
+    record: &mut inverted_index::RSIndexResult,
+    delta: u64,
+    wide: bool,
+) -> usize {
+    let mut buffer_writer = BufferWriter::new(&mut buffer.0);
+
+    if wide {
+        unsafe {
+            bindings::encode_freqs_fields_wide(
+                &mut buffer_writer as *const _ as *mut _,
+                delta,
+                record,
+            )
+        }
+    } else {
+        unsafe {
+            bindings::encode_freqs_fields(&mut buffer_writer as *const _ as *mut _, delta, record)
+        }
+    }
+}
+
+pub fn encode_freqs_fields_wide(
+    buffer: &mut TestBuffer,
+    record: &mut inverted_index::RSIndexResult,
+    delta: u64,
+) -> usize {
+    let mut buffer_writer = BufferWriter::new(&mut buffer.0);
+
+    unsafe {
+        bindings::encode_freqs_fields_wide(&mut buffer_writer as *const _ as *mut _, delta, record)
+    }
+}
```

Comment:
What's this one for? I think we only use the above one right?

Reply:
Leftover from one of my many refactoring. :)